api/WindowEventHandlers.json

```
@@ -452,33 +452,9 @@
             "edge": {
               "version_added": "79"
             },
-            "firefox": [
-              {
-                "version_added": "69"
-              },
-              {
-                "version_added": "68",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "dom.promise_rejection_events.enabled",
-                    "value_to_set": "true"
-                  }
-                ]
-              },
-              {
-                "version_added": "55",
-                "partial_implementation": true,
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "dom.promise_rejection_events.enabled",
-                    "value_to_set": "true"
-                  }
-                ],
-                "notes": "This event handler was added in Firefox 55 but was disabled since it wasn't fully implemented. It was fully implemented in Firefox 68 and enabled by default in Firefox 69."
-              }
-            ],
+            "firefox": {
+              "version_added": "69"
+            },
             "firefox_android": {
               "version_added": false
```

I'm pretty sure this is wrong and should be 79:

```suggestion
              "version_added": "79"
```

---

These have been deferred to #15890!